src/a_prep/initialize_matrices.f90

```diff
@@ -36,7 +36,7 @@ subroutine initialize_mats_new
             &scale_z, mesh, nx0, ny0, nz0, weightx, weighty, weightz, meshproc_tot, &
             &volmesh_proc, minz_at, from_ions, rion_from, nx_at, ny_at, nz_at, rion_upload
     use fourier_module, only: ncub_min, ncub_max, nx_proc, nxny_proc, nx8, nxny8&
-            &, ind_init
+            &, ind_init, norm_metric
```

Comment:
Hi @addman2, norm_metric is a subroutine of allio.f90, isn't it?

Reply:
You are right. It is strange because nvidia compiler does not see it. But only that one. So maybe it is a bug of nvidia compiler. It is definatelly a problem for 22.3. I can Try latest 23.1. I will mark this a a draft until I figure out what is going on.

Reply:
The problem persists also with 23.1. I need to investigate more

Reply:
Ok I solved it! take a look. I dont know why Nvidia compiler does not see it, but after I explicitly load it it works.

Reply:
OK! Great. I approve the changes. In the near future, we should add only statement for all the modules...